### PR TITLE
Standardize hardware documentation

### PR DESCRIPTION
**Pull request by tekstrand**
## What
1. Update headings to follow semantic guidelines
2. Shorten some tab names (t-beam)
3. Add missing headings under tabs for semantics
4. Remove extra duplicate words from headings when parent heading already contains context
5. Standardize purchase link formatting for all RAK devices

## Why
1. Semantic html is easier to read, better for accessibility, and better for SEO
2. There were enough tab names and the names were long enough that they were starting to wrap

## Screenshots
Screenshot 2024-01-26 at 1 06 16 PM
Screenshot 2024-01-26 at 1 06 10 PM
Screenshot 2024-01-26 at 1 06 06 PM
Screenshot 2024-01-26 at 1 06 02 PM
Screenshot 2024-01-26 at 1 05 59 PM

### CODE REVIEW

**Comment by pdxlocations** [comment]
@tekstrand These edits look great!  Would you mind signing the CLA?

**Comment by tekstrand** [comment]
Ah that is a real bummer. A result of using tabs unfortunately. Definitely do not merge this as is - we need to either add them back and standardize that or create a fix for the tabs.

**Comment by tekstrand** [comment]
@rcarteraz thoughts on this solution? https://docusaurus.io/docs/api/themes/configuration#table-of-contents

Screenshot 2024-01-25 at 6 48 25 PM

**Comment by tekstrand** [comment]
Realized that the default config worked so reverted back, however did convert the indentation in that file over to spaces. If I need to remove that commit just lmk.

File: docs/hardware/devices/heltec/index.mdx
Comment (pdxlocations) [inline docs/hardware/devices/heltec/index.mdx:93]: This purchase link header was added for V3 but is missing from Heltecs WSL, Tracker, & Paper.

Also missing from all T-Beams other than Supreme

File: docs/hardware/devices/heltec/index.mdx
Comment (pdxlocations) [inline docs/hardware/devices/heltec/index.mdx:101]: IMO this would be more clear if labeled "HELTEC WSL V3"  if not "HELTEC Wireless Stick Lite V3"

We could eventually add the Wireless Stick (non lite)  as supported so for consistency it would need to be in the header.

File: docs/hardware/devices/lora/index.mdx
Comment (pdxlocations) [inline docs/hardware/devices/lora/index.mdx:141]: v2.1-1.6

File: docs/hardware/devices/nano/index.mdx
Comment (pdxlocations) [inline docs/hardware/devices/nano/index.mdx:27]: "Nano G2 Ultra"

File: docs/hardware/devices/rak/base-boards.mdx
Comment (pdxlocations) [inline docs/hardware/devices/rak/base-boards.mdx:86]: Would "international" make more sense than "overseas"?

File: docs/hardware/devices/rak/base-boards.mdx
Comment (pdxlocations) [inline docs/hardware/devices/rak/base-boards.mdx:116]: RAK isn't a US-based distributor, perhaps "international distributor"

File: docs/hardware/devices/rak/base-boards.mdx
Comment (pdxlocations) [inline docs/hardware/devices/rak/base-boards.mdx:153]: Not US

File: docs/hardware/devices/tbeam/index.mdx
Comment (pdxlocations) [inline docs/hardware/devices/tbeam/index.mdx:105]: "SX1276"

File: docs/hardware/devices/tbeam/index.mdx
Comment (pdxlocations) [inline docs/hardware/devices/tbeam/index.mdx:140]: capital "SX"

**Review by pdxlocations — CHANGES_REQUESTED** [review]
I love the new look of the TOC!  The links never actually worked unless you're in the tab already so that's still a bummer.

File: docs/hardware/devices/heltec/index.mdx
Comment (tekstrand) [reply docs/hardware/devices/heltec/index.mdx:93]: I can follow up with updates here, or remove these updates from the current. Any preference? I do think there are other updates to standardize beyond what we've done here and are talking about. Wanted to focus on headings mostly in this PR.

File: docs/hardware/devices/rak/base-boards.mdx
Comment (tekstrand) [reply docs/hardware/devices/rak/base-boards.mdx]: Changed to international, if we feel strongly about dist/manu I can change that too. Just because you're the manu doesn't mean you aren't a dist though.

**Comment by tekstrand** [comment]
♻️ updated with feedback

Tabs/TOC is an interesting topic. There's an [issue that was closed](https://github.com/facebook/docusaurus/issues/5343) by the maintainer of docusaurus requesting the feature. Amazed at the attitude that guy is responding with but that is beside the point.

I know we are considering a move to a different docs platform. I am not sure how many users we have that are visually impaired or using screen reading devices, but we should be sure consider those users as we choose a new platform and make UX decisions around doc structure.

File: docs/hardware/devices/heltec/index.mdx
Comment (pdxlocations) [reply docs/hardware/devices/heltec/index.mdx:93]: I like the new format you came up with, just given the PR title I thought maybe you skipped the others accidentally.

File: docs/hardware/devices/rak/base-boards.mdx
Comment (pdxlocations) [reply docs/hardware/devices/rak/base-boards.mdx]: Could just say "US" and "international" and skip "Distributor"  Other links just say "AliExpress" however.  Coming up with a standard would be ideal.

**Comment by pdxlocations** [comment]
> I know we are considering a move to a different docs platform. I am not sure how many users we have that are visually impaired or using screen reading devices, but we should be sure consider those users as we choose a new platform and make UX decisions around doc structure.

I second this.

File: docs/hardware/devices/heltec/index.mdx
Comment (tekstrand) [reply docs/hardware/devices/heltec/index.mdx:93]: Thank you. Not an accidental miss, I just didn't want to go too deep in on my first contribution in case it was all shot down anyways.  I plan to follow up with more PRs.

File: docs/hardware/devices/rak/base-boards.mdx
Comment (tekstrand) [reply docs/hardware/devices/rak/base-boards.mdx]: Thoughts on this?

```
### Resources
 - Resource 1
 - Resource n
 - Purchase links
   - US
     - Purchase link 1
     - Purchase link 2
   - International
     - Purchase link 1
     - Purchase link 2
```

Comment (pdxlocations) [reply docs/hardware/devices/rak/base-boards.mdx]: Yeah that works, I'm not picky at all about the form as long as it's consistent.  I think @rcarteraz made most or all of these pages so I'll defer to his ultimate wisdom.

**Comment by tekstrand** [comment]
♻️ purchase links standardized across all RAK pages. Will follow up with PRs for other devices but starting with RAK here.

File: docs/hardware/devices/rak/peripherals.mdx
Comment (pdxlocations) [inline docs/hardware/devices/rak/peripherals.mdx:42]: I hate being so picky but there's an extra space "RAK1910"

Its funny because the content of this page is wrong, but it's the extra space that bugs. me.

**Review by pdxlocations — APPROVED** [review]

**Comment by tekstrand** [comment]
We'll call you hawkeye from now on! 😂 
hawkeye

**Comment by pdxlocations** [comment]
I think this is ready to ship, unless @rcarteraz has any suggestions?

**Comment by tekstrand** [comment]
Sounds good, I'll start working on the other sections once this is merged.

**Pull request merged**
